contributors/design-proposals/predicates-ordering.md

```diff
@@ -0,0 +1,90 @@
+# predicates ordering
+
+
+
+status: proposal
+
+Author: Yassine TIJANI.
+Approvers: 
+* gmarek
+* bsalamat
+* k82cn
+
+
+
+
+## Abstract
+
+This document describes how and why reordering predicates helps to achieve performance for the kubernetes scheduler. We will expose the motivations behind this proposal, The two steps/solution we see to tackle this problem and the timeline decided to implement these.
```

Comment:
There's general suggestion that: build paragraph multiple lines, so reviewers can comments on each line here.